src/WooCommerce.php

```diff
@@ -107,6 +107,11 @@ protected function isWooCommerceTemplate(string $template): bool
     protected function locateThemeTemplate(string $template): string
     {
         $themeTemplate = WC()->template_path() . str_replace(\WC_ABSPATH . 'templates/', '', $template);
+        
+        if(is_child_theme()) {
```

Comment:
Tiny nitpick to follow the conventions of this repo and since I’m on my phone, could you add a space between the if token and the opening parentheses `if (`

Reply:
Solved with the second commit. :)